src/main/features/dashboard/routes/claim-details.ts

```diff
@@ -0,0 +1,24 @@
+import * as express from 'express'
+import { Paths as DashboardPaths } from 'dashboard/paths'
+import ClaimStoreClient from 'claims/claimStoreClient'
+import Claim from 'app/claims/models/claim'
+import ErrorHandling from 'common/errorHandling'
+
+async function renderView (req: express.Request, res: express.Response): Promise<void> {
+  const claim: Claim = await ClaimStoreClient.retrieveByClaimReference(res.locals.user.dashboardDraft.document.search.reference, res.locals.user.bearerToken)
+  res.render(DashboardPaths.claimDetailsPage.associatedView, {
+    claimNumber: claim.claimNumber,
+    partyStripeValue: claim.claimNumber,
+    claimantVDefendant: claim.claimData.claimants[0].name + ' v ' + claim.claimData.defendants[0].name,
```

Comment:
use string template?
```
${claim.claimData.claimants[0].name} v ${claim.claimData.defendants[0].name}
```

I would consider adding a getter to claimData which retrieves the first claimant / defendant
That would make this:
```
${claim.claimData.primaryClaimant.name} v ${claim.claimData.primaryDefendants.name}
```

Reply:
done